projects/storefrontlib/cms-components/product/product-list/product-facet-navigation/facet/facet.component.ts

```diff
@@ -53,6 +57,16 @@ export class FacetComponent {
     return this._facet;
   }
 
+  @Optional() featureConfigService = inject(FeatureConfigService, {
+    optional: true,
+  });
+
+  get isFacetKeyboardNavigationEnabled(): boolean {
```

Comment:
Interesting way to have a feature flag in an attribute. I think we definitely want a comment here to clean this up and maybe even a `@deprecated` message so its clear that its not meant to be a public api.

Reply:
Comment added. I can't use `@deprecated` or SonarCube complains at me for using deprecated code. 🙈

Reply:
Ah I see. Thanks for adding the comment then :D